packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -70,10 +72,24 @@ export const FormControl = (props: FormControlProps) => {
           hasError={hasError}
           isDisabled={isDisabled}
           isRequired={isRequired}
-          sx={{ marginRight: 'auto' }}
+          sx={{ display: 'flex', alignItems: 'center' }}
         >
           {typeof label === 'string' ? label : undefined}
         </FormLabel>
+        {icon && (
+          // TODO: This is a temporary fix. Replace this when the tooltip component is introduced
```

Comment:
❓ Do we have a PR for the tooltip component?

Reply:
@nikosdouvlis it is not a PR yet, but we do have a ticket for it